### ISSUE TITLE
Unable to access all options of Settings pane

### ISSUE THREAD

**Issue by PARAG00991** [gambitproject/jsgte#63]
There is no way in which we can access the save and reset button of settings pane displays when we click settings button present on toolbar. The screenshot that would justify this issue --
![screenshot from 2016-03-14 19 51 53](https://cloud.githubusercontent.com/assets/8044561/13747610/15e66878-ea1f-11e5-9267-2c4206ac7572.png)
This is the view of the complete window, and we can't go down in the Settings pane.

**Comment by hvk3** [comment]
Are you sure that you haven't magnified your browser or something? This is in Chrome, without any alterations.
![test](https://cloud.githubusercontent.com/assets/14114822/13749607/3ed2ff7c-ea28-11e5-86ec-de7315666a30.png)

**Comment by PARAG00991** [comment]
@hvk3  yeah, I'm sure about this issue. I'm using firefox on linux OS. Even you can see that my browser is not magnified, showing 100% as size.
![20160314_212141](https://cloud.githubusercontent.com/assets/8044561/13750703/a3ffec9e-ea2c-11e5-83ee-d4ab935307ae.jpg)
Sorry for the poor quality of image.

**Comment by hvk3** [comment]
Might be a browser-specific issue then, because Chrome has no such problems.

Edit : Definitely a browser-specific problem. Your change is causing problems in Chrome, by shifting 9 player colors in one line, and one remaining color in a new line.
